input/pagecontent/future.md

```diff
@@ -34,8 +34,18 @@ The approach to develop and mature AU Core is outlined below making reference to
 ### Future Candidate Requirements Under Consideration
 The following items are under consideration to add to AU Core.
 
-#### AU Core MedicationStatement 
-Development and testing of AU Core MedicationStatement profile to support AUCDI Medication Use Statement is planned for the AU Core R2 scope.
-
-#### Last Updated 
-AUCDI element Last Updated is intended to reflect when a data group was last updated, capturing clinical relevance and currency. A known and implemented solution in other jurisdictions for similar requirements is the use of the Provenance resource. An alternative to the use of Provenance may be the definition of custom extensions. Future versions of AU Core will develop and test approaches to addressing Last Updated.
\ No newline at end of file
+* **Last Updated**: AUCDI element Last Updated is intended to reflect when a data group was last updated, capturing clinical relevance and currency. A known and implemented solution in other jurisdictions for similar requirements is the use of the Provenance resource. An alternative to the use of Provenance may be the definition of custom extensions. Future versions of AU Core will develop and test approaches to addressing Last Updated.
+* **AU Core MedicationStatement**: Development and testing of AU Core MedicationStatement profile to support AUCDI Medication Use Statement is planned for the AU Core R2 scope.
+* **Dosage**: The Dosage data type is complex, and the requirements in AU Base Dosage are not sufficient to provide clarity on the minimum requirements for supporting dosage and timing in Australia. Future versions of AU Core will develop and test an AU Core Dosage profile to support common implementation of medicine information. 
+* **Composition**: Future versions of AU Core will develop and test AU Core Composition profile to support approaches to sharing clinical documents across multiple use cases.
+* **DocumentReference**: Future versions of AU Core will develop and test approaches and AU Core DocumentReference profile to support sharing of clinical notes across multiple use cases to support common implementation.
```

Comment:
Suggest: "Future versions of AU Core will develop and test approaches **to** AU Core DocumentReference profile..."

Reply:
agree wording change is needed but we don't want to 'to a profile'. we want to develop and test a profile.

Reply:
fixed wording